src/main/java/org/rumbledb/types/ItemType.java

```diff
@@ -217,4 +217,46 @@ public String toString() {
         }
         throw new OurBadException("Unrecognized type.");
     }
+
+    public static String convertClassNameToItemTypeName(String fullTypeName) {
```

Comment:
I think the parameter should be renamed to className here.

Furthermore, I believe the variable in line 222 was defined to use within the fullTypeName.endsWith(....) call. 

Lastly, how about we use the [uncapitalize() function](http://commons.apache.org/proper/commons-lang//apidocs/org/apache/commons/lang3/StringUtils.html#uncapitalize-java.lang.String-) from apache commons? I believe this would make the body of the if statement more readable.